mapper-processor/src/main/java/com/datastax/oss/driver/internal/mapper/processor/util/generation/GeneratedCodePatterns.java

```diff
@@ -259,6 +292,113 @@ public static void setValue(
+      BindableHandlingSharedCode enclosingClass,
+      CodeBlock nullSavingStrategy) {
+    generateSetWithNullSavingStrategy(
+        cqlName,
+        valueExtractor,
+        targetName,
+        methodBuilder,
+        nullSavingStrategy,
+        CodeBlock.of(
+            "$1L = $1L.set($2L, $3L, $4L)",
+            targetName,
+            cqlName,
+            valueExtractor,
+            enclosingClass.addGenericTypeConstant(typeName)));
+  }
+
+  private static void generateSetWithClass(
+      CodeBlock cqlName,
+      CodeBlock valueExtractor,
+      String targetName,
+      MethodSpec.Builder methodBuilder,
+      TypeName typeName,
+      CodeBlock nullSavingStrategy) {
+
+    generateSetWithNullSavingStrategy(
+        cqlName,
+        valueExtractor,
+        targetName,
+        methodBuilder,
+        nullSavingStrategy,
+        CodeBlock.of(
+            "$1L = $1L.set($2L, $3L, $4T.class)", targetName, cqlName, valueExtractor, typeName));
+  }
+
+  private static void generatePrimitiveSet(
+      CodeBlock cqlName,
+      CodeBlock valueExtractor,
+      String targetName,
+      MethodSpec.Builder methodBuilder,
+      String primitiveAccessor,
+      CodeBlock nullSavingStrategy) {
+
+    // todo how to handle primitive type getters? (int id == null is not possible)
+    //    generateSetWithNullSavingStrategy(cqlName, valueExtractor, targetName, methodBuilder,
+    // nullSavingStrategy,
+    //        CodeBlock.of(
+    //            "$1L = $1L.set$2L($3L, $4L)", targetName, primitiveAccessor, cqlName,
+    // valueExtractor));
+    methodBuilder.addStatement(
+        "$1L = $1L.set$2L($3L, $4L)",
+        targetName,
+        primitiveAccessor,
+        cqlName,
+        valueExtractor); // ignore nullSaving strategy for now
+  }
+
+  private static void generateSetWithNullSavingStrategy(
+      CodeBlock cqlName,
+      CodeBlock valueExtractor,
+      String targetName,
+      MethodSpec.Builder methodBuilder,
+      CodeBlock nullSavingStrategy,
+      CodeBlock nonNullStatement) {
+    if (nullSavingStrategy.equals(DO_NOT_SET_CODE_BLOCK)) {
+      methodBuilder.beginControlFlow("if($1L == null)", valueExtractor);
+      methodBuilder.addComment("do not set this field");
+      methodBuilder.endControlFlow();
+      methodBuilder.beginControlFlow("else");
+      methodBuilder.addStatement(nonNullStatement);
+      methodBuilder.endControlFlow();
+    } else if (nullSavingStrategy.equals(SET_TO_NULL_CODE_BLOCK)) {
+      methodBuilder.beginControlFlow("if($1L == null)", valueExtractor);
+      methodBuilder.addStatement("$1L = $1L.setToNull($2L)", targetName, cqlName);
+      methodBuilder.endControlFlow();
+      methodBuilder.beginControlFlow("else");
+      methodBuilder.addStatement(nonNullStatement);
+      methodBuilder.endControlFlow();
+    } else {
+      // the nullSavingStrategy is dynamic - we cannot infer it at compile time
+      methodBuilder.beginControlFlow("if($1L == null)", valueExtractor);
```

Comment:
Could you add a space after `if` in the generated code? That's how it's formatted everywhere else.
```suggestion
      methodBuilder.beginControlFlow("if ($1L == null)", valueExtractor);
```